GPU/Common/FramebufferManagerCommon.cpp

```diff
@@ -1957,7 +1959,7 @@ void FramebufferManagerCommon::UpdateFramebufUsage(VirtualFramebuffer *vfb) {
 
 	checkFlag(FB_USAGE_DISPLAYED_FRAMEBUFFER, vfb->last_frame_displayed);
 	checkFlag(FB_USAGE_TEXTURE, vfb->last_frame_used);
-	checkFlag(FB_USAGE_RENDERTARGET, vfb->last_frame_render);
+	checkFlag(FB_USAGE_RENDER_COLOR, vfb->last_frame_render);
```

Comment:
Hm, what if it's only rendering depth?  I could see it being useful to know that... hm.  Maybe for render-to-texture selection too.  Can't remember if I've seen games render a scene with RGB and stencil all masked out, just to establish depth... feel like I have...

-[Unknown]

Reply:
Yeah, this tracking isn't perfect yet. I don't think I have seen such a case though, but I also don't think it'll hurt very much if we get that wrong..